packages/formation/__tests__/js/accordion/accordion.test.js

```diff
@@ -103,6 +103,14 @@ document.body.innerHTML = `
 `;
 
 describe('accordion', () => {
+  it('should set aria-expanded attribute on load if non exists', () => {
+    require('../../../js');
```

Comment:
I haven't seen a `require` inside an `it()`. Does this work differently than just importing at the top of the file?

Reply:
@ncksllvn just checked and I thought I needed this to reload on each test but just loading it after the DOM was created worked. Thanks. I think I got the first test working then just copy pasted it and didn't realize I had it was still there.